lib/pf/UnifiedApi/Controller/Config/Subtype.pm

```diff
@@ -0,0 +1,68 @@
+package pf::UnifiedApi::Controller::Config::Subtype;
+
+=head1 NAME
+
+pf::UnifiedApi::Controller::Config::Subtype -
+
+=cut
+
+=head1 DESCRIPTION
+
+pf::UnifiedApi::Controller::Config::Subtype
+
+=cut
+
+use strict;
+use warnings;
+use Mojo::Base qw(pf::UnifiedApi::Controller::Config);
+
+sub form_class_by_type {
+    my ($self, $type) = @_;
+    return undef;
+}
+
+sub form {
```

Comment:
The logic of this subroutine seems to be duplicated in the actual controller implementations, could you confirm it is really needed?